skin/build.gradle

```diff
@@ -173,3 +191,6 @@ bintray {
         publish = true
     }
 }
+repositories {
```

Comment:
I think we could put this in the top level `build.gradle`'s allprojects repositories block

Reply:
Fixing this